ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Week 7: Git Challenge
+
+Welcome to week 7 of software training! This week, you'll be completing a series of `git` and GitHub challenges. **To get started, type your GT username into the Title box above, then click the "Submit new issue" at the bottom of this page!** From there, work through the rest of this checklist.
+
+If you need any help, we recommend these online resources:
+- https://help.github.com/articles/github-glossary/
+- https://try.github.io/levels/1/challenges/1
+- https://learngitbranching.js.org/
+- https://chris.beams.io/posts/git-commit/
+
+You can, of course, also flag down an instructor if you run into anything.
+
+## Your objectives
+
+Replace `gburdell3` with your GT username anywhere it appears in this list.
+
+- [ ] Star this repository.
+- [ ] Create a fork of this repository.
+- [ ] Make a copy of that repository that lives on your computer instead of on a GitHub server somewhere. Don’t forget, this version of the repo is connected to the remote version so that changes can be synced between the two.
```

Comment:
I think that this should be easy to understand or removed, see comments on fork above.